kbase-extension/static/kbase/js/common/cellComponents/tabs/results/reportWidget.js

```diff
@@ -55,7 +55,8 @@ define(['bluebird', 'jquery', 'common/html', 'common/ui', 'common/events', 'kbas
             } else {
                 const reportContainer = document.createElement('div');
                 toggleHeader.parentElement.appendChild(reportContainer);
-                new KBaseReportView($(reportContainer), { report_ref: objInfo.reportRef });
+                new KBaseReportView($(reportContainer), { report_ref: objInfo.reportRef })
+                    .loadAndRender();
```

Comment:
rendering is a separate step now, so it can return a Promise that can be consumed, where appropriate.